frontend/src/metabase/visualizations/components/ChartSettingsWidget.jsx

```diff
@@ -32,20 +30,7 @@ const ChartSettingsWidget = ({
         disable: disabled,
       })}
     >
-      {title && (
-        <h4 className="mb1 flex align-center">
-          {title}
-          <Icon
-            size={12}
-            className={cx("ml1 text-light text-medium-hover cursor-pointer", {
-              hidden: !set || noReset,
```

Comment:
It looks like `set` and `noReset` aren't needed in this component anymore. They could at least be dropped from the destructured props and I'm guessing `noReset` can be deleted upstream too.

Reply:
Thanks @paulrosenzweig. I've attempted doing that in [this latest commit](https://github.com/metabase/metabase/pull/13968/commits/4355ad5a23a0235365b062aa802f82cc5bd6355d) — does it seem right?